templates/default/elastic.yaml.erb

```diff
@@ -7,6 +7,9 @@ instances:
     <% unless i["pshard_stats"].nil? %>
     pshard_stats: <%= i["pshard_stats"] %>
     <% end %>
+    <% unless i["cluster_stats"].nil? %>
+      cluster_stats: <%= i["cluster_stats"] %>
```

Comment:
the indentation is not correct here: `cluster_stats` needs to have the same indentation as the other parameters. Could you correct it?

(same comment for the `ssl`  options of `mongo.yaml.erb`)